Sources/RunnerLib/Files Import/DangerFileGenerator.swift

```diff
@@ -10,7 +10,7 @@ public final class DangerFileGenerator {
 
         importsRegex.enumerateMatches(in: content, options: [], range: NSRange(location: 0, length: content.count), using: { result, _, _ in
```

Comment:
<!--
  0 failure: 
  1 warning:  Line should be 12...
  
  
  DangerID: danger-id-default;
  File: Sources/RunnerLib/Files Import/DangerFileGenerator.swift;
  Line: 11;
-->

- :warning: Line should be 120 characters or less: currently 140 characters (`line_length`)